CrackingTheCodingInterview/Chapter1ArraysAndStrings/Permutation.java

```diff
@@ -22,4 +28,54 @@ public boolean permutation(String s, String t) {
 
 		return true;
 	}
-}
\ No newline at end of file
+	
+	/**
+	*boolean method to compare two int arrays and determine whether they are permutations of each other.
+	@param two int arrays 
+	*Assume there are no duplicates
+	*/
+	public static boolean areIntArrayPermutations1(int[] a1, int[] a2){
```

Comment:
This method is unnecessary as it handles a specific edge case when there are no duplicates. Should try to solve the most general case.

Please remove.